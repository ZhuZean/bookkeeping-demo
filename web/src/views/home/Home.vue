<template>
  <div class="home">
    <div id="home-container" class="container">
      <div class="row">
        <div class="col">
          <Summary></Summary>
        </div>
        <div class="col ml-5">
          <add-record></add-record>
          <hr class="mt-3 mb-3" style="border: 0; border-top: 2px solid #FFC107;"/>
          <bill-detail class="mt-4"></bill-detail>
        </div>
      </div>
    </div>
  </div>
</template>

<script lang="ts">
import Summary from "./components/Summary.vue"
import AddRecord from "./components/AddRecord.vue"
import BillDetail from "./components/BillDetail.vue"

export default {
  name: "Home",
  components: {
    Summary,
    AddRecord,
    BillDetail
  },
};
</script>

<style lang="scss">
#home-container {
  margin-top: 10em;
}
</style>
