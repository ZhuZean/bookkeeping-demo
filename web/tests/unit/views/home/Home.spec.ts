import { shallowMount } from '@vue/test-utils'
import Home from '@/views/home/Home.vue'

describe('Home.vue', () => {
  it('can render', () => {
    const wrapper = shallowMount(Home)
    expect(wrapper.element).toMatchSnapshot()
  })
})
