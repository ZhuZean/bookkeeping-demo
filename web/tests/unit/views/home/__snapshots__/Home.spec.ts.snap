// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue can render 1`] = `
<div
  class="home"
>
  <div
    class="container"
    id="home-container"
  >
    <div
      class="row"
    >
      <div
        class="col"
      >
        <bill-summary-stub />
      </div>
      <div
        class="col ml-5"
      >
        <add-record-stub />
        <hr
          class="mt-3 mb-3"
          style="border: 0px; border-top: 2px solid #FFC107;"
        />
        <bill-detail-stub
          class="mt-4"
        />
      </div>
    </div>
  </div>
</div>
`;
